test/doctool/test-doctool-html.js

```diff
@@ -115,18 +115,19 @@ testData.forEach((item) => {
           assert.ifError(err);
 
           const actual = output.replace(spaces, '');
+          const scriptDomain = 'google-analytics.com'
           // Assert that the input stripped of all whitespace contains the
           // expected list
           assert(actual.includes(expected));
 
           // Testing the insertion of Google Analytics script when
           // an analytics id is provided. Should not be present by default
           if (includeAnalytics) {
-            assert(actual.includes('google-analytics.com'),
-                   'Google Analytics script was not present');
+            assert(actual.includes(scriptDomain),
+                   'Google Analytics script was not present, but it should be');
```

Comment:
The idea here is to include the actual value and not only to change the message.
So it should actually be like
```js
assert(actual.includes(scriptDomain),
           `Google Analytics script was not present in "${actual}"`);
```

Reply:
@chowdhurian could you push a commit to this branch that makes the change @BridgeAR suggested? LGTM with those changes.